kyuubi-common/src/main/scala/org/apache/kyuubi/config/KyuubiConf.scala

```diff
@@ -3473,7 +3473,7 @@ object KyuubiConf {
         "after `spark-submit` is returned.")
       .version("1.7.1")
       .timeConf
-      .createWithDefaultString("PT30S")
+      .createWithDefaultString("PT5M")
```

Comment:
It is too short now, we shall enlarge it for production.

Reply:
I think the default value great than `kyuubi.session.engine.initialize.timeout` (3 min) does not make sense